tower-balance/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![deny(dead_code)]
```

Comment:
I tend to just put `#![deny(warnings]` in all libraries, it catches plenty of mistakes. Well, a slightly bigger list:

```rust
#![deny(warnings)]
#![deny(missing_docs)]
#![deny(missing_debug_implementations)]
```